stories/data-viewer.js

```diff
@@ -1,44 +1,41 @@
-import {storiesOf} from '@storybook/vue';
+import { storiesOf } from '@storybook/vue';
 
-import {DataViewer} from '../dist/storybook/components';
+import { DataViewer } from '../dist/storybook/components';
 
 const stories = storiesOf('DataViewer', module);
 
 stories.add('empty', () => ({
-                       components: {DataViewer},
-                       template: `
+  store: setupStore({}, [], true),
+  components: { DataViewer },
+  template: `
     <data-viewer>
     </data-viewer>
   `,
-                     }));
+}));
 
 stories.add('simple', () => ({
-                        components: {DataViewer},
-                        props: {
-                          dataset: {
-                            default() {
-                              return {
-                                headers:
-                                    [
-                                      {name: 'columnA'},
-                                      {name: 'columnB'},
-                                      {name: 'columnC'},
-                                    ],
-                                    data: [
-                                      ['value1', 'value2', 'value3'],
-                                      ['value4', 'value5', 'value6'],
-                                      ['value7', 'value8', 'value9'],
-                                      ['value10', 'value11', 'value12'],
-                                      ['value10', {obj: 'value14'}, null],
-                                    ]
-                              }
-                            },
-                          },
-                        },
-                        template: `
+  store: setupStore({
+    dataset: {
+      headers:
+        [
+          { name: 'columnA' },
+          { name: 'columnB' },
+          { name: 'columnC' },
+        ],
+      data: [
+        ['value1', 'value2', 'value3'],
+        ['value4', 'value5', 'value6'],
+        ['value7', 'value8', 'value9'],
+        ['value10', 'value11', 'value12'],
+        ['value10', { obj: 'value14' }, null],
+      ]
+    },
+  }, [], false),
```

Comment:
Why is autobind `false` and why specify it ? It looks like it's the default value

Reply:
good catch, it's a mistake.